public/usage-examples/graphics/current_clip/current-clip-1-simple.cpp

```diff
@@ -0,0 +1,24 @@
+#include "splashkit.h"
+
+int main() 
+{
+    // Open a window with the title "Current Clip" and dimensions
+    window win = open_window("Current Clip", 800, 600);
+
+    // Define the clipping area
+    rectangle rect = current_clip(win);
+
+    // Draw the clipping information
+    draw_text("Current Clip: " + std::to_string(rect.width) + " X " + std::to_string(rect.height), COLOR_BLACK, "Arial", 24, 100, 100);
+    
+    // Refresh the screen to display the text
+    refresh_screen();
+
+    // Wait for 5 seconds
+    delay(5000);
+
+    // Close all windows
```

Comment:
Make sure the commits are the same as well, so it makes less confusion to the users.